php/class-wp-seo-settings.php

```diff
@@ -162,6 +171,32 @@ public function set_properties() {
 		 * @param  array Associative array of setting names and values.
 		 */
 		$this->default_options = apply_filters( 'wp_seo_default_options', array( 'post_types' => array_keys( $this->single_post_types ), 'taxonomies' => array_keys( $this->taxonomies ) ) );
+
+		/**
+		 * Checks database to see if taxonomy migration has been run.
+		 *
+		 * @since v0.13.0
+		 */
+		$this->taxonomy_migration = get_option( $this::SLUG . '-taxonomy-migration', false );
```

Comment:
A new option seems a little heavy, especially if we update something else about the option structure again later. 

Could we store the status in our existing option?

Thinking out loud, we could add a top-level `internal` key to the option array and make it clear that the data are for core plugin use only. Then our check could be like `if ( empty( $option['internal']['migrated_taxonomies_to_taxonomy_key'] ) )...`.